chore: align sending flow for stickers with the new sending flow

saledjenic opened the pull request:
Based on changes done in this PR https://github.com/status-im/status-go/pull/5807 we can simplify our client logic a lot.

This results in the removal of many lines of code that are no longer needed

Closes 1st part of #16336

https://github.com/user-attachments/assets/a2639ef4-4338-4851-b38f-bd0ecab458df

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | ded1408b | [#2](https://ci.status.im/job/status-desktop/job/prs/job/macos/job/aarch64/job/package/job/PR-16398/2/) | 2024-09-25 13:05:07 | ~6 min | `macos/aarch64` | [:apple:`dmg`](https://status-im-desktop-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Desktop-240925-125846-ded140-pr16398-aarch64.dmg) |

caybro reviewed (commented):
The UI tests failures (in SwapModal!) look legit:

```
[2024-09-25T13:23:30.226Z] 3: FAIL!  : QmlTests::SwapModal::test_modal_swap_proposal_setup() 'wait for signal fetchSuggestedRoutesCalled' returned FALSE. ()
[2024-09-25T13:23:30.226Z] 3:    Loc: [/home/jenkins/workspace/s_linux_x86_64_tests-ui_PR-16398/storybook/qmlTests/tests/tst_SwapModal.qml(589)]
```

and

```
[2024-09-25T13:23:41.035Z] 3: FAIL!  : QmlTests::SwapModal::test_no_auto_refresh_when_proposalLoading_or_approvalPending() property count
[2024-09-25T13:23:41.035Z] 3:    Actual   (): 1
[2024-09-25T13:23:41.035Z] 3:    Expected (): 2
[2024-09-25T13:23:41.035Z] 3:    Loc: [/home/jenkins/workspace/s_linux_x86_64_tests-ui_PR-16398/storybook/qmlTests/tests/tst_SwapModal.qml(1851)]
```

status-im-auto commented:
#### :heavy_check_mark: [status-desktop/prs/windows/x86_64/package/PR-16398#3](https://ci.status.im/job/status-desktop/job/prs/job/windows/job/x86_64/job/package/job/PR-16398/3/) :small_blue_diamond: ~20 min :small_blue_diamond: 49029717 :small_blue_diamond: [:package: windows/x86_64 package](https://status-im-desktop-prs.ams3.digitaloceanspaces.com/StatusIm-Desktop-240925-131239-490297-pr16398-x86_64.exe)

virginiabalducci reviewed (approved):
Looks good to me
tested purchasing stickers 
wallet https://sepolia.etherscan.io/address/0x21934ee1626e6fc30af951db1184c687904763aa
tx hash https://sepolia.etherscan.io/tx/0xd3a09e5cab968368070abfa3613deddabbdfe9b50f0a63d3835350c9eca680c5

saledjenic commented:
@caybro since this PR is based on this one https://github.com/status-im/status-desktop/pull/16330 and tests were failing there I fixed them there and just rebased this PR, should be good now. Thanks!

status-im-auto commented:
#### :heavy_check_mark: [status-desktop/prs/windows/x86_64/package/PR-16398#4](https://ci.status.im/job/status-desktop/job/prs/job/windows/job/x86_64/job/package/job/PR-16398/4/) :small_blue_diamond: ~19 min :small_blue_diamond: 2608f440 :small_blue_diamond: [:package: windows/x86_64 package](https://status-im-desktop-prs.ams3.digitaloceanspaces.com/StatusIm-Desktop-240926-082601-2608f4-pr16398-x86_64.exe)

caybro reviewed (approved):
LGTM

status-im-auto commented:
#### :heavy_check_mark: [status-desktop/prs/windows/x86_64/package/PR-16398#5](https://ci.status.im/job/status-desktop/job/prs/job/windows/job/x86_64/job/package/job/PR-16398/5/) :small_blue_diamond: ~19 min :small_blue_diamond: da2a3021 :small_blue_diamond: [:package: windows/x86_64 package](https://status-im-desktop-prs.ams3.digitaloceanspaces.com/StatusIm-Desktop-241001-071129-da2a30-pr16398-x86_64.exe)

status-im-auto commented:
#### :heavy_check_mark: [status-desktop/prs/windows/x86_64/package/PR-16398#6](https://ci.status.im/job/status-desktop/job/prs/job/windows/job/x86_64/job/package/job/PR-16398/6/) :small_blue_diamond: ~18 min :small_blue_diamond: b257f24c :small_blue_diamond: [:package: windows/x86_64 package](https://status-im-desktop-prs.ams3.digitaloceanspaces.com/StatusIm-Desktop-241001-080112-b257f2-pr16398-x86_64.exe)
